examples/spine/src/app.coffee

```diff
@@ -57,23 +57,25 @@ class TodoApp extends Spine.Controller
 		@addNew todo for todo in @getByFilter()
 
 	toggleAll: (e) ->
+		checked = e.target.checked
 		Todo.each (todo) ->
 			###
 			TODO: Model updateAttribute sometimes won't stick:
 				https://github.com/maccman/spine/issues/219
 			###
-			todo.updateAttribute 'completed', e.target.checked
+			todo.updateAttribute 'completed', checked
 			todo.trigger 'update', todo
 
 	clearCompletedItem: ->
 		Todo.destroyCompleted()
 
 	toggleElems: =>
-		isTodos = !!Todo.count()
-		@main.toggle isTodos
-		@footer.toggle isTodos
-		@clearCompleted.toggle !!Todo.completed().length
-		@toggleAllElem.removeAttr 'checked' if !Todo.completed().length
+		dones = Todo.completed().length
```

Comment:
as much as I like the word dones :) perhaps completed might be nicer

:goat: :pizza: